modules/ROOT/pages/_partials/install-mule-runtime-versions.adoc

```diff
@@ -0,0 +1,64 @@
+= Install Mule Runtime Engine in Studio
+
+// tag::intro[]
+Anypoint Studio bundles the latest Runtime engine version available for each release.
+
+If you choose to use a different Mule runtime engine version than the one bundled in your Studio distribution, you can install any compatible Mule runtime version.
+// end::intro[]
+
+// tag::prereq[]
+= Prerequisites
+
+* If you have strict firewall rules, ensure that you allow the following URLs:
+** `+http://studio.mulesoft.org/+`
+** `+http://repository.mulesoft.org/connectors/releases/+`
+** `+http://download.eclipse.org/technology/m2e/releases+`
+** `+http://download.eclipse.org/eclipse/updates+`
+** `+http://subclipse.tigris.org/+`
+** `+http://findbugs.cs.umd.edu/eclipse/+`
+// end::prereq[]
+
+// tag::install-task[]
+= Installing a Different Mule Runtime Engine Version
```

Comment:
For a task with steps, use imperative: Install.
To introduce a set of task topics or for an action without steps, use the gerund Installing.
```suggestion
== Install a Different Mule Runtime Engine Version
```